core-ui/src/hooks/useMessageList.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import LuigiClient from '@luigi-project/client';
+import { Link } from 'fundamental-react';
+import { Dropdown } from 'react-shared';
+import { useTranslation } from 'react-i18next';
+
+export const EVENT_MESSAGE_TYPE = {
+  ALL: { type: 'All', label: 'all' },
+  NORMAL: { type: 'Normal', label: 'normal' },
+  WARNING: { type: 'Warning', label: 'warnings' },
+};
+
+export const RESOURCE_PATH = {
+  Pod: 'pods',
```

Comment:
Would it be (easily) possible to handle `Node`s here, too?

Reply:
Nodes have a separate column at the cluster/node view, but added them anyway.